src/Umbraco.Cms.Api.Management/Controllers/Document/ByKeyDocumentController.cs

```diff
@@ -22,30 +22,6 @@ public class ByKeyDocumentController : DocumentControllerBase
     private readonly IDocumentPresentationFactory _documentPresentationFactory;
     private readonly IContentQueryService _contentQueryService;
 
-    [Obsolete("Scheduled for removal in v17")]
-    public ByKeyDocumentController(
-        IAuthorizationService authorizationService,
-        IContentEditingService contentEditingService,
-        IDocumentPresentationFactory documentPresentationFactory)
-    {
-        _authorizationService = authorizationService;
-        _documentPresentationFactory = documentPresentationFactory;
-        _contentQueryService = StaticServiceProvider.Instance.GetRequiredService<IContentQueryService>();
-    }
-
-    // needed for greedy selection until other constructor remains in v17
-    [Obsolete("Scheduled for removal in v17")]
-    public ByKeyDocumentController(
-        IAuthorizationService authorizationService,
-        IContentEditingService contentEditingService,
-        IDocumentPresentationFactory documentPresentationFactory,
-        IContentQueryService contentQueryService)
-    {
-        _authorizationService = authorizationService;
-        _documentPresentationFactory = documentPresentationFactory;
-        _contentQueryService = contentQueryService;
-    }
-
     [ActivatorUtilitiesConstructor]
```

Comment:
Can remove this when there's only one constructor left.  See also other controller constructors where you have removed obsolete ones to leave only one remaining.